centos/content.md

```diff
@@ -26,7 +26,9 @@ By default, the CentOS containers are built using yum's `nodocs` option, which h
 
 # Systemd integration
 
-Currently, systemd in CentOS 7 has been removed and replaced with a `fakesystemd` package for dependency resolution. This is due to systemd requiring the `CAP_SYS_ADMIN` capability, as well as being able to read the host's cgroups. If you wish to replace the fakesystemd package and use systemd normally, please follow the steps below.
+Systemd is now included in both the centos:7 and centos:latest base
+containers, but it is not active by default. In order to use systemd, you will
+need to include text similar to the example Dockerfile below:
```

Comment:
Can you make these one line; that is what the Travis build is complaining about: https://travis-ci.org/docker-library/docs/builds/100199255#L155-L165.